cmd/main_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+	"regexp"
+	"strconv"
+	"testing"
+
+	"github.com/SUMUKHA-PK/Raft-Distributed-Consensus/types"
+)
+
+func TestMain(t *testing.T) {
+	//check if the config file exists
+	info, err := os.Stat("../server.config.json")
+	if os.IsNotExist(err) {
```

Comment:
In all places where theres an error, let following be the format.
`t.Fatalf("whatever you want to say : %v",err)